### PR TITLE
Update demos to libcanard v3 and cyphalize

### PR DESCRIPTION
**Pull request by pavel-kirienko**
New `socketcan.[ch]` are copy-pasted from https://github.com/OpenCyphal-Garage/platform_specific_components

@coderkalyan I can't tag you for review because you are not a member of this team yet (sent the invite just now)

### CODE REVIEW

**Comment by coderkalyan** [comment]
Could you check/send the invite again? I don't think I got it...

**Comment by pavel-kirienko** [comment]
@coderkalyan done. Please visit https://github.com/OpenCyphal-Garage to accept it.

**Comment by coderkalyan** [comment]
Got it, thanks.

File: differential_pressure_sensor/README.md [hunk 1]
```diff
@@ -62,8 +62,8 @@ Next, we launch a PnP node-ID allocator available in Yakut (PX4 also implements
 
 ```bash
 export UAVCAN__CAN__IFACE="socketcan:vcan0"
-export UAVCAN__NODE__ID=127                 # This node-ID is for Yakut.
-yakut monitor -P ~/allocation_table.db
+export UAVCAN__NODE__ID=127                     # This node-ID is for Yakut.
+y mon --plug-and-play ~/allocation_table.db
```
Comment (coderkalyan) [inline differential_pressure_sensor/README.md:66]: Where was this abbreviation aliased? Might be better to keep it expanded considering this is a README?

Comment (coderkalyan) [reply differential_pressure_sensor/README.md:66]: Hmm I guess it's used a lot and is fine, so never mind.

**Review by coderkalyan — APPROVED** [review]
LGTM code wise. I'd still prefer not to have command aliases in READMEs but that's a small nitpick which doesn't matter.

File: differential_pressure_sensor/README.md (hunk 1 above)
Comment (pavel-kirienko) [reply differential_pressure_sensor/README.md:66]: "Y A K U T" is verbose -- who has the time to type all these letters anyway?

**Pull request merged**